utils.py

```diff
@@ -27,6 +28,10 @@ def getCVE_Dict(year: str) -> {}:
     :return: CVE dictionary
     """
     cve_dict = {}
+    if 'nvd' not in listdir():
```

Comment:
InstalledSoftware: open_element_by_key - instead of printing, use a logger (python has a built in logger). this way will allow to customize the logging (to a file / print)